techniques/css/C34.html

```diff
@@ -207,28 +203,28 @@ <h3>Example 1: Un-fix sticky headers / footers in HTML and CSS</h3>
 
 
 <section id="tests">
-      <h2>Tests</h2>
+      <h2>検証</h2>
       <section class="test-procedure" id="procedure">
-        <h3>Procedure</h3>
+        <h3>手順</h3>
         <div class="note">
-				<p class="note-title marker">Note</p>
-				<p>Depending on the environment to be tested the actual modes or sizes can differ.</p>
+				<p class="note-title marker">注記</p>
+				<p>実際のモード又はサイズは、テストする環境によって異なる。</p>
 			</div>
         <ol>
-          <li>Display content on a device / user agent in portrait mode.</li>
-          <li>Change the orientation to landscape.</li>
-          <li>Check whether the sticky header and footer will be un-fixed depending on the existing media query settings.</li>
-          <li>Display content on a desktop / user agent at a starting viewport width of 1280x1024 CSS pixels.</li>
-          <li>Change the viewport size in width and height or use the zoom function of the browser.</li>
-          <li>Check whether the sticky header and footer will be un-fixed at specific sizes depending on the existing media query settings.</li>
+          <li>デバイス/ユーザエージェント上のコンテンツを縦向きモードで表示する。</li>
+          <li>方向を横向きに変更する。</li>
+          <li>既存のメディアクエリーの設定に応じて、スティッキーヘッダー及びスティッキーフッターが固定解除されるかどうかをチェックする。</li>
+          <li>>開始時のビューポートの幅を 1280x1024 CSS ピクセルとし、デスクトップ/ユーザエージェントにコンテンツを表示する。</li>
```

Comment:
`<li>` タグの山括弧が重複していました。
```suggestion
          <li>開始時のビューポートの幅を 1280x1024 CSS ピクセルとし、デスクトップ/ユーザエージェントにコンテンツを表示する。</li>
```